### DISCUSSION TITLE
I am trying to make people have homes

### DISCUSSION THREAD

**Discussion by ghost** [EssentialsX/Essentials#4658] [Q&A] [answered]
Hi I want everyone to have multiple homes, I am only using essentials and they can make at least 1 home but no more. is there any way to fix if so can I get some help?

**Comment by Chew** [comment, marked as answer]
Do they have `essentials.sethome.multiple` and the groups (`essentials.sethome.multiple.[set name]`)?

**Reply by ghost** [reply]
Um I am really not sure, do you want my config file?
